### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Thomasalex2**
- Done with P0 and P1 points of the component library

### CODE REVIEW

File: resources/fervent-ui/Components/cards.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+.card-layout:hover {
+    box-shadow: rgba(0, 0, 0, 0.15) 4px 4px 4px;
+    transition: box-shadow 0.2s ease-in-out;
+}
+
+.card-header h4 {
+    font-weight: bold;
+    margin: 0.5rem 1rem;
+    margin-bottom:0.1rem;
```
Comment (ritikaawasthy) [inline resources/fervent-ui/Components/cards.css]: you could use px here for values smaller than 10px

File: resources/fervent-ui/Components/dialogue.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    gap: 3rem;
+    margin: 3rem;
+    align-items: center;
+}
+
+.dlg-box {
+    /* position: absolute;
+    top:50%;
+    left:50%;
+    transform: translate(-50%, -50%); */
```
Comment (ritikaawasthy) [inline resources/fervent-ui/Components/dialogue.css]: you can remove unwanted comments

**Review by ritikaawasthy — CHANGES_REQUESTED** [review]
just some minor changes otherwise it looks great, UI is very clean

File: resources/fervent-ui/Components/cards.css (hunk 1 above)
Comment (Thomasalex2) [reply resources/fervent-ui/Components/cards.css]: Thanks, yes. I will change it to px

File: resources/fervent-ui/Components/dialogue.css (hunk 2 above)
Comment (Thomasalex2) [reply resources/fervent-ui/Components/dialogue.css]: Ahh, I missed that. Thank you so much for reviewing!

**Pull request merged**